webpack.config.native.js

```diff
@@ -1,22 +1,15 @@
+// This webpack config outputs only the **native JS bundle**.
+// The results is placed in the "dist" folder, with each file carrying a "bundle.native.js" suffix. 
+// It is separated into its own webpack file, so that the webpack-dev-server
+//    doesn't try to inline additional code into it, which would only work
+//    on the web but not under the native execution environment.
+
+const CommonConfig = require("./webpack.common");
+
 module.exports = {
-  entry: {
-    "customfunctions": "./src/customfunctions/functions.ts"
-  },
+  ...CommonConfig,
   output: {
-    filename: '[name].bundle.native.js',
-    path: __dirname + '/dist'
-  },
-  devtool: "source-map",
-  resolve: {
-    extensions: [".ts", ".js"]
-  },
-  module: {
-    rules: [
-      {
-        test: /\.tsx?$/,
-        exclude: /node_modules/,
-        use: "ts-loader"
-      }
-    ]
-  },
+    filename: "[name].bundle.native.js",
```

Comment:
"native" is not the best word here as it implies native compiled code. For ReactNative, the pattern is "index.{platform}.bundle" with platform being "win32" for Windows. The platform is required. We should rename "native" back to "win32".